app/src/main/java/guessme/ppc2017/unnoba/edu/ar/gameone/Renderer.java

```diff
@@ -0,0 +1,174 @@
+package guessme.ppc2017.unnoba.edu.ar.gameone;
+
+import android.app.Activity;
+import android.graphics.Typeface;
+import android.widget.TextView;
+
+import java.util.ArrayList;
+
+/**
+ * Created by anton on 9/5/2017.
+ */
+
+public class Renderer {
+    //initial numbers that must be displayed.
+    private TextView number1;
```

Comment:
Not sure, but I think that all these TextViews could be inside an Array, then you can set them individually in linkElementsWithView(...), but you will be able to set them the font in a for in realRender(...)